docs/references/go/overview.mdx

```diff
@@ -75,24 +83,28 @@ for _, resource := range list.$Resource$s {
 }
 ```
 
+</InjectKeys>
+
 ### Usage with a client
 
-If you're working with multiple keys, you should use the library with a client. The API packages for each
+If you're working with multiple keys, it's recommended to use Clerk Go with a client. The API packages for each
 resource export a `Client`, which supports all the API's operations.
-This way you can create as many clients as needed, each with their own API key, as shown in the following example:
+This way, you can create as many clients as needed, each with their own API key, as shown in the following example:
+
+<InjectKeys>
 
 ```go
 import (
-    "github.com/clerk/clerk-sdk-go/v2"
-    "github.com/clerk/clerk-sdk-go/v2/$resource$"
+  "github.com/clerk/clerk-sdk-go/v2"
+  "github.com/clerk/clerk-sdk-go/v2/$resource$"
 )
 
 // Each operation requires a context.Context as the first argument.
 ctx := context.Background()
 
 // Initialize a client with an API key
 config := &clerk.ClientConfig{}
-config.Key = "sk_live_XXX"
+config.Key = "{{secret_key}}"
```

Comment:
Same here about this snippet being pseudo-code in case we don't want to show the actual keys. Feel free to resolve the discussion if we do want that. :smile: